src/protocolgame.cpp

```diff
@@ -2692,7 +2692,7 @@ void ProtocolGame::sendOutfitWindow()
 		}
 
 		protocolOutfits.emplace_back(outfit.name, outfit.lookType, addons);
-		if (protocolOutfits.size() == 100) { // Game client doesn't allow more than 100 outfits
+		if (protocolOutfits.size() == 255) { // FIXME: Figure out the current limit
```

Comment:
So if it's 255, just change the comment back to what it was saying originally, write 255 in it instead and it should be ready to merge.

Reply:
done